proto/api_v3/query_service.proto

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2021 The Jaeger Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax="proto3";
+
+package jaeger.api_v3;
+
+import "opentelemetry/proto/trace/v1/trace.proto";
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/duration.proto";
+
+option go_package = "api_v3";
+option java_package = "io.jaegertracing.api_v3";
+
+// Request object to get a trace.
+message GetTraceRequest {
+  // Hex encoded 128 bit trace_id
+  string trace_id = 1;
+}
+
+// A single response chunk holds spans from a single trace.
+// Note that a subsequent request to get a trace might return a different result
+// because spans are reported in asynchronous chunks.
```

Comment:
> because spans are reported in asynchronous chunks

not sure what this means. There are two reasons why the responses could be different: (1) new spans arrived since the first request, and (2) the order of span loading from storage is not deterministic / guaranteed.

Reply:
maybe it's better to remove this comment